react/components/Slider/index.js

```diff
@@ -39,17 +39,19 @@ export default class SlickSlider extends Component {
     leftArrowClasses: PropTypes.string,
     /** Right arrow custom classes */
     rightArrowClasses: PropTypes.string,
+    /** Dots custom classes */
+    dotsClasses: PropTypes.string,
   }
 
   getSettings(slideWidth) {
-    const { sliderSettings, adaptToScreen, scrollByPage, defaultItemWidth, children, leftArrowClasses, rightArrowClasses } = this.props
+    const { sliderSettings, adaptToScreen, scrollByPage, defaultItemWidth, children, leftArrowClasses, rightArrowClasses, dotsClasses } = this.props
     const itemsPerPage = getItemsPerPage(this._slick, slideWidth, defaultItemWidth, sliderSettings.slidesToShow)
     const settings = { ...sliderSettings }
     const numItems = children.length
 
     settings.nextArrow = settings.nextArrow || <Arrow customClasses={rightArrowClasses} cssClass={VTEXClasses.ARROW_RIGHT_CLASS} />
     settings.prevArrow = settings.prevArrow || <Arrow customClasses={leftArrowClasses} cssClass={VTEXClasses.ARROW_LEFT_CLASS} />
-    settings.appendDots = dots => <Dots dots={dots} cssClass={VTEXClasses.DOTS_CLASS} />
+    settings.appendDots = dots => <Dots dots={dots} dotsClasses={dotsClasses} cssClass={VTEXClasses.DOTS_CLASS} />
```

Comment:
`cssClass={VTEXClasses.DOTS_CLASS}`. Why the need for another prop, since they are both injected in the same element?

Reply:
Ok, I'm not gonna keep holding this PR, but we should change that later, since the naming is becoming very confusing for this props and values!